### DISCUSSION TITLE
Should I also add Paid Food ( Friday / Sunday )  Menu  ??

### DISCUSSION THREAD

**Discussion by Dipankar-Kumar-Singh** [Dipankar-Kumar-Singh/AIT_MESS_Telegram_Bot#4] [Ideas]
I feel that, I often have to decide at Mess what to take this time , I think I should have pre decided , but due to lack of Menu , I still have to decide there , by observation , I feel : 
This **shows down the order taking process** ( for both mess - Khanna mess + Thapa mess ) 
Having a look at menu before reaching mess can speedup the process and clear the line faster.  

But not sure if people will need it or not on real ground . 😅
should I add that feature ? 
@generalpy101 what do you suggest ?  🤔

**Comment by generalpy101** [comment]
Why not

It might be a good addition
